src/main/java/com/example/school/model/Teacher.java

```diff
@@ -0,0 +1,21 @@
+package com.example.school.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+import static java.util.UUID.randomUUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class Teacher {
+    @Builder.Default
+    private UUID id = randomUUID();
+    private String name;
+    private String surname;
```

Comment:
It would be better to add `startDate` field, when teacher starts working in the school.